support/additional-prebuilts-to-delete.txt

```diff
@@ -0,0 +1,40 @@
+# this is a list of packages that end up in the tarball's prebuilts folder but are not actually needed.
```

Comment:
Will this list need to be updated for 3.1?

Reply:
Yes, but I was going to wait until we hit zero in the baseline, I think that will be easier.